src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IWithStatement.vb

```diff
@@ -112,5 +112,43 @@ IBlockOperation (3 statements) (OperationKind.Block, Type: null) (Syntax: 'Priva
 
             VerifyOperationTreeAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub IWithStatement_StaticFieldReferenceAsRValue_NoInstanceReceiver()
+            Dim source = <![CDATA[
+Imports System
+
+Structure SSS
+    Public Shared A As String
+
+    Public Sub New(_a As String)
+    End Sub
+End Structure
+
+Class Clazz
+    Sub TEST(i As Integer)
+        With New SSS(Me.ToString())
+            Static s As Action = Sub()
```

Comment:
>Static s As Action = Sub() [](start = 12, length = 26)

It looks like Static variable and the lambda are not required to test this scenario.

Reply:
Removed.

---
In reply to: [149579913](https://github.com/dotnet/roslyn/pull/22972#discussion_r149579913) [](ancestors = 149579913)